docs/appguide.rst

```diff
@@ -80,37 +80,37 @@ any change in its lifecycle.
         GO_TO_CLOCK [ label="GOTO ClockApp" ];
     }
 
-When an application is initialized is enters the ``BACKGROUND`` state. A
+When an application is initialized, it enters the ``BACKGROUND`` state. A
 backgrounded application will not execute but it should nevertheless
 maintain its user visible state whilst deactivated. To conserve
 memory wasp-os does not permit two applications to run simultaneously but
-because each application remembers its state when it is not running then it
+because each application remembers its state when it is not running, then it
```

Comment:
This fix doesn't seem right.

This is a paranthetic clause so there should also be a comma after the but and the "then" should be removed after "running,".